extensions/video-call-reactions/package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "video-call-reactions",
+  "title": "Video Call Reactions",
+  "description": "This extension triggers video call reactions such as hearts, thumbs up/down, balloons, rain, confetti, fireworks, and lasers on macOS.",
+  "icon": "video-call-reactions-icon.png",
+  "author": "yazan_zaid",
+  "categories": [
+    "Communication",
+    "Applications",
+    "Fun",
+    "Media",
+    "Other",
+    "Productivity",
+    "System",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Trigger Video Reaction",
```

Comment:
I would maybe add an [argument](https://developers.raycast.com/information/lifecycle/arguments) to change it from the default one?

Reply:
I've included both the 'favorite reaction' and 'custom reaction' options, allowing users to swiftly toggle between two distinct choices. It's unclear to me whether the arguments retain their status upon closing or toggling, or if I need to implement this feature myself. 😊

Reply:
What about simply having just one command and use the arguments placeholder with the placeholder `Default Action` and then let the user switch to another?

Reply:
> What about simply having just one command and use the arguments placeholder with the placeholder `Default Action` and then let the user switch to another?

Can the arguments save last state?